{"uuid": "fd19225b-56b7-45c7-864e-f6f32a64b937", "children": ["29613ad6-40b6-443d-b10b-e1b0e6a22f08"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Failed to decode response from marionette\n\n", "trace": "  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"S:\\IT\\Spriint_6-main\\conftest.py\", line 9, in driver\n    driver = webdriver.Firefox()\n             ^^^^^^^^^^^^^^^^^^^\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 71, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 241, in __init__\n    self.start_session(capabilities)\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 329, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"S:\\IT\\Spriint_6-main\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1736396395157, "stop": 1736396408729}], "start": 1736396395157, "stop": 1736396425193}